app/viewTests/shared/suites/SecvisogramPage/EditWorkflowStateDialog.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import EditWorkflowStateDialog from '../../../../lib/app/SecvisogramPage/DocumentsTab/View/EditWorkflowStateDialog.js'
+
+export const title = 'SecvisogramPage-EditWorkflowStateDialog'
+
+const props = {
+  onSubmit() {},
+  data: {
+    advisoryId: 'my-advisory',
+    allowedStateChanges: ['Draft', 'Published'],
```

Comment:
Since this is just test data for a visual UI test, it doesn't matter here. But addressed it in https://github.com/secvisogram/secvisogram/pull/324/commits/31d47eb417b0e53abb6cf5f10cba8795c7bfb1e3